tests/signal/test_hampel_filter.py

```diff
@@ -0,0 +1,64 @@
+"""Test_hampel_filter.
+
+- Author: Sunjin.kim
+- Contact: sunjin.kim@onepredict.com
+"""
+
+import numpy as np
+from numpy.testing import assert_array_almost_equal
+
+from onebone.signal import hampel_filter
+
+
+def timeseries_data(num_outlier: int = 10) -> np.ndarray:
+
+    """
+    make example timeseries
+    You can set the number of random outliers.
+
+    Parameters
+    ----------
+    num_outlier : numpy.ndarray
+        You can set the number of random outliers.
+        default = 10
+    """
+    t = np.linspace(0, 1, 1000)
+    y = np.sin(2 * np.pi * 10 * t)
+    make_outlier = np.random.randint(0, 1000, size=num_outlier)
+    np.put(y, [make_outlier], 10)
+
+    return y
+
+
+def test_hampel_filter(is_plot: bool = False):
+    window_size = 4
+    filtered_data = hampel_filter.hampel_filter(timeseries_data(), window_size)[0]
+    origin_data = timeseries_data()
+    perfect_data = timeseries_data(num_outlier=0)
```

Comment:
1. 순서만 바꾸면 함수 호출 횟수를 줄일 수 있습니다.
2. 노이즈가 없는 데이터는 `origin_data`로 노이즈가 있는 변수는 `noisy_data`가 좋을 것 같습니다.
```suggestion
    origin_data = timeseries_data(num_outlier=0)
    noisy_data = timeseries_data(num_outlier=10)
    filtered_data = hampel_filter.hampel_filter(noisy_data, window_size)[0]

```

Reply:
> 1. 순서만 바꾸면 함수 호출 횟수를 줄일 수 있습니다.
> 2. 노이즈가 없는 데이터는 `origin_data`로 노이즈가 있는 변수는 `noisy_data`가 좋을 것 같습니다.

감사합니다. 다만, 이름을 정할때 기존 origin data (노이즈가 존재하는 데이터) 를 hampel filter 를  통해 filter 시키면 노이즈가 제거된 데이터가 되게 되며, 이걸 비교할 지표를 찾다가 임의적으로 ideal 한 데이터를 만들어 비교하기 위해 perfect data 로 설정하였습니다. 

의도가 위와 같다보니 노이즈가 없는 데이터를 origin data 로 설정하는게 조금 고민이됩니다
우선은 말씀해주신대로 수정해놓았습니다!